debian8/guide/services/ssh/ssh_server/sshd_disable_root_login.rule

```diff
@@ -23,4 +23,5 @@ references:
     anssi: NT007(R21)
     nist: AC-3,AC-6(2),IA-2(1)
 
-ocil: '<sshd-check-macro default="no" option="PermitRootLogin" value="no" />'
+
+{{{ sshd_check_complete_ocil_entry(default="no", option="PermitRootLogin", value="no") }}}
```

Comment:
This seems odd to me. I don't think that people who are used to Ansible and OpenControl would find this familiar either. I understand the purpose and reasoning, but I would expect this:

```
ocil: {{{ sshd_check_complete_ocil_entry(default="no", option="PermitRootLogin", value="no") }}}
```

Reply:
Also there is an added whitespace for no reason.

Reply:
Have you checked out the full definition of the macro? There was an issue that an ocil clause has to be added as well, and the `complete_ocil_entry` refers to adding `ocil` as well as the corresponding `ocil_clause`. As the clause should be added every time, it made sense to me to make a encompassing macro, so there are no omissions.

Reply:
I have also removed those whitespaces.